packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -172,3 +180,17 @@ export function setValueForProperty(
     }
   }
 }
+
+// Only used in DEV, stringifies a value and Warns if it took too long
+const stringifyWithPerformanceWarning = value => {
+  const stringifyStart = Date.now();
```

Comment:
We want to use `performance.now` if it exists, and fallback to `Date.now` if it doesn't. Lets do the same thing we do here: https://github.com/facebook/react/blob/master/packages/react-native-renderer/src/ReactNativeFrameScheduling.js#L12-L17

Reply:
Thanks a lot for the review, I'll get on it sometime soon!